### PR TITLE
mapping hotfixes + balances and adjustments

### PR DESCRIPTION
**Pull request by karlodayo**
+ colony size adjustment (smaller)
+ colony/ishimura mapping
+ rig additions/balances + price balances
+ weapon/ammo price balances
+ marker spawn adjustments

### CODE REVIEW

**Comment by nianoru** [comment]
you ever just make a million commits that only fixes :trollface:

**Review by DTraitor — APPROVED** [review]

**Comment by Viz-Titan** [comment]
image

File: code/modules/research/designs/design_weapons.dm [hunk 1]
```diff
@@ -277,15 +277,15 @@ S	fuel tank (hydrazine)
 	id = "javelin_rack"
 	materials = list(MATERIAL_PLASTEEL = 4000, MATERIAL_PHORON = 2500)
 	build_path = /obj/item/ammo_magazine/javelin
-	price = 2000	// 400 per 2 spears, same as in DS2
+	price = 2800	// 400 per 2 spears, same as in DS2
```
Comment (Viz-Titan) [inline code/modules/research/designs/design_weapons.dm]: Also get rid of this comment

File: html/changelogs/nianorufix.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+delete-after: True
+
+changes:
+  - balance: "Colony: adjusted trash-dump marker spawn location in sewers."
+  - balance: "Colony: lower medical now requires a brute/exploder in the old closet area to break through."
+  - feature: "Colony: added pipe dispensers to engineering."
+  - feature: "Colony: added ladder near excavation site."
+  - feature: "Colony: added ladder to the right of science (in maints)."
+  - feature: "Colony: the SCAF shuttle now requires an underground entry only, filled with land mines."
+  - balance: "Colony: adjusted trash-dump marker spawn location in sewers."
```
Comment (Viz-Titan) [inline html/changelogs/nianorufix.yml]: Said the same thing twice about trash-dump

**Review by DTraitor — APPROVED** [review]

**Comment by nianoru** [comment]
Adding one final commit to this PR, then you can close/merge it... I think we have enough PRs going on right now.

**Comment by nianoru** [comment]
+ evac revamp soon, dont merge PR yet, after that, im done.

**Comment by nianoru** [comment]
+ evac revamp soon, dont merge PR yet, after that, im done.

**Comment by nianoru** [comment]
if there is no problem with anything regarding maps this weekend, this PR can be closed and merged now whenever you guys want.

**Comment by nianoru** [comment]
this version of evac does not include windows that show outside into space yet; im trying to figure out how to make a new wall type in the meantime, but this is still usable.

**Comment by nianoru** [comment]
this version of evac does not include windows that show outside into space yet; im trying to figure out how to make a new wall type in the meantime, but this is still usable.

**Comment by karlodayo** [comment]
this version of evac does not include windows that show outside into space yet; im trying to figure out how to make a new wall type in the meantime, but this is still usable and ready for the weekend. 

feel free to merge this PR into the codebase whenever ready, im basically done with it.

**Pull request merged**